crates/router/src/connector/adyen/transformers.rs

```diff
@@ -4087,19 +4071,41 @@ pub struct AdyenPayoutCancelRequest {
     merchant_account: Secret<String>,
 }
 
+#[cfg(feature = "payouts")]
+impl TryFrom<&PayoutMethodData> for PayoutCardDetails {
+    type Error = Error;
+    fn try_from(item: &PayoutMethodData) -> Result<Self, Self::Error> {
+        match item {
+            PayoutMethodData::Card(card) => Ok(Self {
+                payment_method_type: "scheme".to_string(), // FIXME: Remove hardcoding
+                number: card.card_number.peek().to_string(),
+                expiry_month: card.expiry_month.peek().to_string(),
+                expiry_year: card.expiry_year.peek().to_string(),
+                holder_name: card
```

Comment:
@kashif-m Are you using dynamic fields for payouts? if so please make changes in `crates/router/src/configs/defaults.rs`